### DISCUSSION TITLE
Looking for advisory (paid consultancy)

### DISCUSSION THREAD

**Discussion by phpia** [UMEP-dev/UMEP#494] [General]
Hi I work in a spain company an we are lloking for paid consultancy about using UMEP with a large dataset

Thanks

**Comment by torm89** [comment]
Can you provide more details or any contact information?

**Reply by biglimp** [reply]
Not in Spain but I know some people in Sweden. Also people in Italy.

**Reply by torm89** [reply]
I see. You can share my email address with them. My email address is contact@torm.ml.

**Reply by phpia** [reply]
Hi @biglimp  @torm89 could you please send me an email to dpardo AT idelista.com ? And will continue by email?

Thanks a lot

**Reply by torm89** [reply]
@phpia Yes, I'll sent you an email but isn't a typo in the email's domain name? Domain `idelista.com` is not taken.

**Reply by phpia** [reply]
Yep, sorry.    dpardo AT **idealista.com**
